package.json

```diff
@@ -35,13 +35,13 @@
     "clean": "rm -rf lib/ dist/ coverage/ .nyc_output/ jsdoc/",
     "docs": "yarn build:docs && jsdoc -c .jsdoc.json --verbose",
     "test": "yarn test:node && yarn test:integration && yarn test:browser",
-    "test:node": "yarn build:test && yarn _nyc mocha --recursive 'test/unit/**/*.js'",
-    "test:integration": "yarn _nyc mocha --recursive 'test/integration/**/*.js'",
+    "test:node": "yarn _build && yarn build:test && yarn _nyc mocha --recursive 'test/unit/**/*.js'",
```

Comment:
noticed locally that running `yarn test:node` often wasn't running latest .ts changes, need to rebuild .ts first

Reply:
yeah I did this separation on purpose so that you can iterate on tests faster (you don't need to `yarn build` if you're just changing tests)

Reply:
yep, figured there was nuances, ok, not sure on which is best dx, if you change any app ts code you have to yarn build manually first before running any test targets or you'll get stale/incorrect results, I'll revert unless you feel it's worth changing.

Reply:
yeah it depends on your dev flow, too - it's just nice to run tests quickly w/o needing to build. maybe there should be a warning about it being stale? I dunno, but agreed that it's not ideal. reverting sounds good since we both know the quirks now

Reply:
reverted - https://github.com/stellar/js-soroban-client/pull/161/commits/1f483db99ce2c45d566b97d10858cb32418f800f